src/main/java/lotto/domain/winning/BonusMatchStatus.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain.winning;
+
+public enum BonusMatchStatus {
+    MATCHED {
+        @Override
+        public boolean matches(boolean bonusMatch) {
+            return bonusMatch == true;
+        }
+    },
```

Comment:
```suggestion
    MATCHED {
        @Override
        public boolean matches(boolean bonusMatch) {
            return bonusMatch;
        }
    },
```
바로 bonusMatch를 반환해도 괜찮을 것 같아요!

Reply:
아하!! 인텔리제이도 자동으로 위와같이 할 수 있다고 알려주더라구요!
하지만 저는 뭔가 살짝 이해도를 떨어뜨리는 느낌인 거 같았어요..!

확실하게 true여야 하는지 false여야 하는지를 반환해주는게 코드를 더 잘 이해할 수 있게 해준다고 생각했습니다 ㅎㅎㅎ

좋은 의견 감사합니다!!